### DISCUSSION TITLE
Internal & External routing for Application GW and AGIC

### DISCUSSION THREAD

**Discussion by Java4all** [Azure/application-gateway-kubernetes-ingress#1187] [Q&A] [answered]
Hi Guys

Maybe it is a simple question, but I am looking for a solution to have two Application Gateway for one AKS cluster to route internal and external traffic. Based on docs I didn't catch that AGIC supports such design. To be more specific - I am looking for multiple ingress support with different AppGWs.

Please, could you drop here some notes, comments, or suggestions on how to deal with such a solution?

**Comment by garvincasimir** [comment, marked as answer]
Check this doc: proposals/multiple-gateways-single-cluster.md

**Reply by Java4all** [reply]
Yes, this is exactly what I need. Looks like your work is already merged to master. What is really need it is details or a guide on how to use it. Thank you @garvincasimir. I am going to test it.
